tests/org.eclipse.n4js.ide.tests/src/org/eclipse/n4js/ide/tests/server/AbstractOrganizeImportsTest.java

```diff
@@ -58,30 +54,6 @@
 					IssueCodes.IMP_UNUSED_IMPORT,
 					IssueCodes.IMP_UNRESOLVED)));
 
-	/** Some default modules that export a number of classes, for use in the main module of organize imports tests. */
-	protected final List<Pair<String, String>> defaultOrganizeImportsTestModules = Lists.newArrayList(
-			Pair.of("A", Strings.fromLines(
-					"export class A01 {}",
-					"export class A02 {}",
-					"export class A03 {}",
-					"export class A04 {}")),
-			Pair.of("B", Strings.fromLines(
-					"export class B01 {}",
-					"export class B02 {}",
-					"export class B03 {}",
-					"export class B04 {}")),
-			Pair.of("C", Strings.fromLines(
-					"export class C01 {}",
-					"export class C02 {}",
-					"export class C03 {}",
-					"export class C04 {}")),
-			Pair.of("Def", Strings.fromLines(
-					"export class Def01 {}",
-					"export class Def02 {}",
-					"export class Def03 {}",
-					"export class Def04 {}",
-					"export default class DefCls {}")));
```

Comment:
Good idea to use an abstract method for this, but please keep these default modules in `AbstractOrganizeImportsTest`, because the idea was to have a set of default modules for all organize imports tests, i.e. for all subclasses. Currently there's only one subclass but I have introduced a second on branch `GH-1691C`.